<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>result</class>
 <widget class="QWidget" name="result">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>940</width>
    <height>540</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="windowIcon">
   <iconset resource="gNAP.qrc">
    <normaloff>:/picture/picture/icon.png</normaloff>:/picture/picture/icon.png</iconset>
  </property>
  <widget class="QFrame" name="back_result">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>940</width>
     <height>540</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">#back_result{
        border-image: url(:/picture/picture/background.jpg);
}</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QPushButton" name="close">
    <property name="geometry">
     <rect>
      <x>880</x>
      <y>20</y>
      <width>40</width>
      <height>40</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">#close{
        border-image: url(:/picture/picture/close1.png);
}

#close:hover{

        border-image: url(:/picture/picture/close2.png);
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="back">
    <property name="geometry">
     <rect>
      <x>830</x>
      <y>20</y>
      <width>38</width>
      <height>38</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">#back{
        border-image: url(:/picture/picture/back1.png);
}

#back:hover{

        border-image: url(:/picture/picture/back2.png);
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QWidget" name="analyze" native="true">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>60</y>
      <width>361</width>
      <height>430</height>
     </rect>
    </property>
    <widget class="QLabel" name="analyze_head">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>10</y>
       <width>160</width>
       <height>50</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">#analyze_head{
        font: 75 28pt &quot;Arial&quot;;
        color:white;
}</string>
     </property>
     <property name="text">
      <string>Analyze</string>
     </property>
    </widget>
    <widget class="QLabel" name="score">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>69</y>
       <width>101</width>
       <height>61</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">#score{
        background-image: url(:/picture/picture/progress.png);
        color:white;
        font: 75 18pt &quot;Arial&quot;;
}
</string>
     </property>
     <property name="text">
      <string>STABLE
SCORE</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
     </property>
    </widget>
    <widget class="QFrame" name="star">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>350</y>
       <width>250</width>
       <height>50</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">#star{

        border-image: url(:/picture/picture/progress.png);
}</string>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
    </widget>
    <widget class="QComboBox" name="choose">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>130</y>
       <width>141</width>
       <height>50</height>
      </rect>
     </property>
     <property name="cursor">
      <cursorShape>IBeamCursor</cursorShape>
     </property>
     <property name="styleSheet">
      <string notr="true">QComboBox::drop-down{
width:0;
}

#choose{

        border-image: url(:/picture/picture/progress.png);
        font: 75 20pt &quot;Comic Sans MS&quot;;
        color:white;

}

#choose:hover{
        border-image: url(:/picture/picture/gray2.png);
}</string>
     </property>
    </widget>
    <widget class="QFrame" name="up_down">
     <property name="geometry">
      <rect>
       <x>310</x>
       <y>130</y>
       <width>51</width>
       <height>51</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">#up_down{
border-image: url(:/picture/picture/to_high.png);}</string>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
    </widget>
    <widget class="QTextBrowser" name="Information">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>180</y>
       <width>361</width>
       <height>191</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">#Information{
        border-image: url(:/picture/picture/progress.png);
        color:white;
        font: 75 18pt &quot;Courier&quot;;
}</string>
     </property>
     <property name="html">
      <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Courier'; font-size:18pt; font-weight:72; font-style:normal;&quot;&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;br /&gt;&lt;/p&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;br /&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
     <property name="cursorWidth">
      <number>0</number>
     </property>
     <property name="openExternalLinks">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QPushButton" name="done">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>130</y>
       <width>161</width>
       <height>51</height>
      </rect>
     </property>
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="styleSheet">
      <string notr="true">#done{
        border-image: url(:/picture/picture/progress.png);
        color:white;
        font: 75 18pt &quot;Arial&quot;;
}
#done:hover{

        border-image: url(:/picture/picture/gray2.png);
}
</string>
     </property>
     <property name="text">
      <string>Show me!</string>
     </property>
    </widget>
    <widget class="QPushButton" name="sbol">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>380</y>
       <width>361</width>
       <height>50</height>
      </rect>
     </property>
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="styleSheet">
      <string notr="true">#sbol{
        border-image: url(:/picture/picture/progress.png);
        color:white;
        font: 75 18pt &quot;Arial&quot;;
}
#sbol:hover{
        border-image: url(:/picture/picture/gray2.png);
}
</string>
     </property>
     <property name="text">
      <string>SBOL  IT !</string>
     </property>
    </widget>
    <widget class="Line" name="line_2">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>110</y>
       <width>251</width>
       <height>20</height>
      </rect>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
    <widget class="QFrame" name="star1">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>70</y>
       <width>45</width>
       <height>45</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border-image: url(:/picture/picture/star.png);</string>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
    </widget>
    <widget class="Line" name="line_4">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>170</y>
       <width>141</width>
       <height>20</height>
      </rect>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
    <widget class="QFrame" name="star2">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>70</y>
       <width>45</width>
       <height>45</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border-image: url(:/picture/picture/star.png);</string>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
    </widget>
    <widget class="QFrame" name="star4">
     <property name="geometry">
      <rect>
       <x>260</x>
       <y>70</y>
       <width>45</width>
       <height>45</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border-image: url(:/picture/picture/star.png);</string>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
    </widget>
    <widget class="QFrame" name="star5">
     <property name="geometry">
      <rect>
       <x>310</x>
       <y>70</y>
       <width>45</width>
       <height>45</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border-image: url(:/picture/picture/star.png);</string>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
    </widget>
    <widget class="QFrame" name="star3">
     <property name="geometry">
      <rect>
       <x>210</x>
       <y>70</y>
       <width>45</width>
       <height>45</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border-image: url(:/picture/picture/star.png);</string>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
    </widget>
    <zorder>analyze_head</zorder>
    <zorder>score</zorder>
    <zorder>star</zorder>
    <zorder>choose</zorder>
    <zorder>done</zorder>
    <zorder>sbol</zorder>
    <zorder>up_down</zorder>
    <zorder>line_2</zorder>
    <zorder>star1</zorder>
    <zorder>line_4</zorder>
    <zorder>star2</zorder>
    <zorder>star4</zorder>
    <zorder>star5</zorder>
    <zorder>star3</zorder>
    <zorder>Information</zorder>
   </widget>
   <widget class="QWidget" name="predict" native="true">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>60</y>
      <width>400</width>
      <height>430</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <widget class="QLabel" name="predict_head">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>10</y>
       <width>171</width>
       <height>51</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">#predict_head{
        font: 75 28pt &quot;Arial&quot;;
        color:white;
}</string>
     </property>
     <property name="text">
      <string>Predict</string>
     </property>
    </widget>
    <widget class="QLabel" name="target">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>80</y>
       <width>191</width>
       <height>50</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">#target{

        border-image: url(:/picture/picture/progress.png);
        color:white;
        font: 75 18pt &quot;Arial&quot;;
}
</string>
     </property>
     <property name="text">
      <string>TARGET</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QTextBrowser" name="target_gene">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>130</y>
       <width>190</width>
       <height>290</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">#target_gene{
        border-image: url(:/picture/picture/progress.png);
color:white;
        font: 75 18pt &quot;Courier&quot;;
}</string>
     </property>
     <property name="html">
      <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Courier'; font-size:18pt; font-weight:72; font-style:normal;&quot;&gt;
&lt;p align=&quot;center&quot; style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;br /&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
    <widget class="QTextBrowser" name="prediction">
     <property name="geometry">
      <rect>
       <x>189</x>
       <y>79</y>
       <width>211</width>
       <height>325</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">#prediction{
        border-image: url(:/picture/picture/progress.png);
color:white;
        font: 75 16pt &quot;Courier&quot;;
}</string>
     </property>
     <property name="html">
      <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Courier'; font-size:16pt; font-weight:72; font-style:normal;&quot;&gt;
&lt;p align=&quot;center&quot; style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;br /&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
    <widget class="Line" name="line_3">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>140</y>
       <width>20</width>
       <height>241</height>
      </rect>
     </property>
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
    </widget>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>760</x>
      <y>470</y>
      <width>171</width>
      <height>57</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url(:/picture/picture/Software-Logo_white_small.png);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
 </widget>
 <resources>
  <include location="gNAP.qrc"/>
 </resources>
 <connections/>
</ui>
